CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ From v1.0.0 onwards, this project adheres to [Semantic Versioning](https://semve
 
 ## Master (Unreleased)
 
-Up to date with releases.
+- Add support for nested test classes
```

Comment:
We should include the PR the feature/bugfix was added in. PR number is sufficient